main/lint.xml

```diff
@@ -26,6 +26,8 @@
     <issue id="MissingTranslation" severity="ignore" />
     <!-- old translation will be deleted by the next crowdin import -->
     <issue id="ExtraTranslation" severity="ignore" />
+    <!-- we will take what is given by crowdin - For locale "fr": "many" is missing currently -->
+    <issue id="MissingQuantity" severity="ignore" />
```

Comment:
Thats strange.
Crowdin follow the unicode plurals specification, which has no "many" for french: https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html

Is this configurable in Lint?

Reply:
> Thats strange. Crowdin follow the unicode plurals specification, which has no "many" for french: https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html#fr

This is not the latest one (CLDR v37 vs. v38).
Please see at https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#fr

> Is this configurable in Lint?

No.
Only using the special lint rule. As we use crowdin for translation, this should not be problematic.

Reply:
Edit: CLDR is currently at version 42.

Reply:
Crowdin knows this issue, see https://community.crowdin.com/t/missing-many-plurals-for-french-language.

The state is:
~~~
We will update all languages where we missing new standards, like the “many” plural form. 
The appropriate epic task has been already created.

Still, it wouldn’t be released soon, because it requires additional investigation.

For example, some users already have French as 100% translated, and adding a new form means they 
will lose this progress. As a result, if their integration is based on the export option “export only 
fully translated file” it wouldn’t work until this new form becomes translated. For those who are fine 
with current forms, this change may quite uncomfortable.

Right now we’re discussing a solution that would work great for all our clients. 
You can sure that the languages will be definitely updated according to new standards.
~~~